cmd/rookflex/cmd/init.go

```diff
@@ -41,8 +41,9 @@ func initPlugin(cmd *cobra.Command, args []string) error {
 		Status: flexvolume.StatusSuccess,
 		Capabilities: &flexvolume.DriverCapabilities{
 			Attach: false,
-			// Required for cephfs (ReadWriteMany)
+			// applying the selinux relabel and fsgroup is too expensive recursively, especially for shared file system
 			SELinuxRelabel: false,
+			FSGroup:        false,
```

Comment:
the `FSGroup` is not exposed in the flex driver capabilities, so we need to find a different way to disable this.